demo/react/components/avatar/default.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { CSSProperties, ReactElement } from 'react';
 
-import { Avatar, Size, Theme } from 'LumX';
+import { Avatar, ButtonEmphasis, IconButton, Size, Theme } from 'LumX';
+import { mdiDelete, mdiEye, mdiPencil } from 'LumX/icons';
 
 const demoContainerStyle: CSSProperties = {
```

Comment:
If the MDX demo is up to date you can remove the TSX demo.
It's an old format of demo that we want to migrate to MDX

Reply:
Ah ! Was wondering about that. Thank, will remove :)